scripts/security/certs-create.sh

```diff
@@ -11,7 +11,16 @@ rm -f *.crt *.csr *_creds *.jks *.srl *.key *.pem *.der *.p12 *.log
 # Generate CA key
 openssl req -new -x509 -keyout snakeoil-ca-1.key -out snakeoil-ca-1.crt -days 365 -subj '/CN=ca1.test.confluentdemo.io/OU=TEST/O=CONFLUENT/L=PaloAlto/ST=Ca/C=US' -passin pass:confluent -passout pass:confluent
 
-users=(kafka1 kafka2 client schemaregistry restproxy connect connectorSA controlcenter ksqlDBserver ksqlDBUser appSA badapp clientListen zookeeper mds)
+# ksqlDB Server (ksqldb-server) and Control Center (control-center) share a certificate; a separate certificate is not generated for ksqldb-server
+# this shared certificate allows the self-signed certificate to be accepted by a Control Center browser user visiting https://localhost:9092 ,
+# without importing and trusting the self-signed CA, and this acceptance will also apply later to Websockets requests to wss://localhost:8089
+# (port-forwarded to ksqldb-server:8089).  This is necessary as browsers never prompt to trust certificates for this kind of wss:// connection,
+# see https://stackoverflow.com/a/23036270/452210 .
```

Comment:
Thanks for adding the StackOverflow link, very helpful to remember the caveat & workaround!